andela/wg-millennium-falcon
# #158283886 Fix migration conflicts on circleci

#### What does this PR do?
Fixes CircleCI migration conflicts

#### Description of Task to be completed?
Fix migration conflicts on CircleCI by deleting older migrations, adding `migrations` to `.gitignore` and auto-migrating.
The python build-pack (https://github.com/Rwothoromo/heroku-buildpack-python/blob/master/bin/compile) was also updated accordingly.

#### How should this be manually tested?
Visit CircleCI and check the build logs for this PR to verify that it runs successfully.
Test the product features on the review app to confirm accordingly.

#### Any background context you want to provide?
The builds were failing because the command `python manage.py makemigrations` on CircleCI was finding pre-existent migrations pushed from earlier git commits.

#### What are the relevant pivotal tracker stories?
[#158283886](https://www.pivotaltracker.com/story/show/158283886)
